files/es/learn/common_questions/thinking_before_coding/index.html

```diff
@@ -36,19 +36,19 @@ <h2 id="Resumen">Resumen</h2>
  <li>¿Qué es necesario hacer, y en qué orden, para alcanzar mis metas?</li>
 </ul>
 
-<p>Todo esto es llamado <em>idealización del proyecto</em> y es un primer paso necesario para alcanzar tus metas, si eres un desarrollador principiante o experimentado.</p>
+<p>Todo esto es llamado <em>idealización del proyecto</em> y es un primer paso necesario para alcanzar tus metas, sin importar si eres un desarrollador principiante o experimentado.</p>
 
 <h2 id="Aprendizaje_Activo">Aprendizaje Activo</h2>
 
 <p><em>No hay aprendizaje activo todavía. <a href="/es/docs/MDN/Comenzando">Por favor, considere la posibilidad de contribuir</a>.</em></p>
```

Comment:
Link roto, esta es la nueva URL.
```suggestion
<p><em>No hay aprendizaje activo todavía. <a href="/es/docs/MDN/Contribute/Getting_started">Por favor, considere la posibilidad de contribuir</a>.</em></p>
```